Gh pages GitHub guide

TsubasaK111 opened the pull request:
@takayukioda review me! 💃

TsubasaK111 committed:
refactor: remove unused redundant image directory

TsubasaK111 committed:
refactor: rewrite 'using github' guide for en/ja

- rewrite the 'using github' guide to fulfill new user stories.

- change links to open in new tabs so that user can easily return to docs

- retake gifs so that they wait for 12secs before starting (so that users can first scan text without distractions)

- update landing and gitter pages in index.md

TsubasaK111 committed:
chore: rebuild static pages

takayukioda commented on source/docs/en/guide_github.md:
```diff
@@ -1,31 +1,60 @@
-Check "Private Repository" if you do not want to publish the repository.
+## Requirements
+- <a href="https://github.com/join" target="_blank"> GitHub account</a>
+- <a href="https://git-scm.com/" target="_blank">Git</a> (If you don't know Git, see <a href="https://try.github.io/" target="_blank">this</a> and <a href="https://git-scm.com/doc" target="_blank">this</a>.)
```
Maybe git-scm link and then github link is a better order.

Alternative option I came up with is,
If you want to know about git, link to git-scm
If you want quick tutorial to use gi, link to github

TsubasaK111 replied:
Hmm...
Here are my thoughts:
1. This should be super short.  
   - I expect that most users who visit this page already are familiar with git.
   - I also expect there will be a minority of people who want to try programming in local, but don't know git/github.
   - Thus, I want this "if you want to know..." component to not break the structure of the "Requirements" section, so it's easier for the git-familiar users to scan.
   - This means the "if you want to know..." component needs to be very short and concise, and occupies 0 rows.
   - That's why it's expressed as "If you don't know Git, see this and this."
2. The easier solution should be listed first.
   - The "git-scm" link is the full official documentation. This is included because official documentation should always be the central source of truth.
   - However, the "github link" is an interactive tutorial, and is far easier to learn git on. It is also an official resource funded by github.
   - Since this tutorial is an official resource AND it's easier, I think users should be directed to this tutorial at first.

TsubasaK111 replied:
TL;DR: I think we should keep it the way it is :P

takayukioda commented on source/docs/en/guide_github.md:
```diff
@@ -1,31 +1,60 @@
+- And of course [Git](https://git-scm.com/).(If you don't know git, see [this](https://try.github.io/) and [this](https://git-scm.com/doc).)  
+- Then, [connect your GitHub account](https://app.code-check.io/settings/social) to code-check.io.
+- Also, get [nodejs/npm](https://nodejs.org/en/download/)
+- And finally, chant `npm install codecheck` in the CLI to summon codecheck. -->
```
Maybe don't need this comment part?

TsubasaK111 replied:
Oh whoops!

TsubasaK111 committed:
refactor: delete unused redundant image resources in the built page.

TsubasaK111 committed:
feat: add .gitignore and remove unnecessary files

takayukioda commented on source/docs/en/guide_github.md:
````diff
@@ -1,31 +1,60 @@
+- The test results will print to stdout like this:
 ```
-$ git push origin master
+$ codecheck
````
syntax color looks not supported, but it might be good to say
"```bash" so we could know it's a bash code / terminal without reading codes

TsubasaK111 replied:
Thanks for the good suggestion!
Unfortunately, I just tried this and I get inconsistent results with syntax coloring, probably because most of the syntax are not standard bash commands. :(
Thus, I think we should stick with the current syntax...

takayukioda commented on source/docs/ja/guide_github.md:
```diff
@@ -1,29 +1,54 @@
+## テストの実行
+- テストする心の準備ができたら、  
+CLIで`codecheck`コマンドを走らせるぞ。
+- そしたらテスト結果が以下のように標準出力されます:  
```
日本語での語尾の統一、難しいっすよね :sweat_smile: (とりあえず治す必要とかは無いです)

TsubasaK111 replied:
難しいっす！気になったのでちょっと治してみました

takayukioda commented:
I leave some notes here :memo: Please take a look ;)
@TsubasaK111 fix me

TsubasaK111 committed:
feat: add google analytics

- add google analytics tracking ID in mkdocs configuration

- this change will cascade to all static pages in the next 'build chore' commit

- also added to .gitignore (preventative medicine ;-) )

TsubasaK111 committed:
refactor: improve guide_github text based on da0shi's feedback

TsubasaK111 committed:
chore: rebuild static pages

TsubasaK111 commented:
@takayukioda review me! ⛵

takayukioda commented:
Sorry, forgot to review
![LGTM](http://lgtm.in/p/ef6UCEcPp)

TsubasaK111 commented:
😍
